src/container/tabs.rs

```diff
@@ -40,9 +40,19 @@ impl Tabs {
         behavior: &mut dyn Behavior<Pane>,
         rect: Rect,
     ) {
+        if let Some(active) = self.active {
+            if !tiles.is_visible(active) {
+                self.active = None;
+            }
+        }
+
         if !self.children.iter().any(|&child| self.is_active(child)) {
             // Make sure something is active:
-            self.active = self.children.first().copied();
+            self.active = self
+                .children
+                .iter()
+                .copied()
+                .find(|&child_id| tiles.is_visible(child_id));
```

Comment:
instead of making the first visible active, it would be _nice_ if, whenever possible, the first visible item left of the currently active would become active.
Concrete: When I make the currently active tab invisible, I the left-most visible becomes active. This feels a bit off, I'd expect a neighboring element to to get the active state.

Reply:
Yeah, I agree that would be preferable, but it requires a bit more work… PRs welcome? :)